docs/OpenID4VPSameDeviceFlow.md

```
@@ -2,105 +2,186 @@
+    - The application requests credential presentation
+
+2. **Authorization Request**: The verifier constructs an authorization request containing the following parameters:
+    * **response_type**: Specifies the type of response expected from the authorization server(wallet) for example in our case response_type = vp_token
+    * **Client_id**: The unique identifier for the client (Verifier) application making the request
+    * **request_uri**: This is a URL where the verifier application directs the Wallet to retrieve the actual Authorization Request, used when the request object is too large to be transmitted directly, helping to keep the Request Object size smaller.
+    * **response_mode**: This parameter tells the wallet that how it should send the vp_token to the verifier application
+
+      | Response Mode         | Description                                                                                                                                                                                                                                                                                                                                                |
+      | --------------------- |------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+      | Direct Post (Default) | Response will be sent to verifier using HTTP POST request and response body will be encoded with content type `application/x-www-form-urlencoded`. <br/> - Response is sent to `response_uri` controlled by the verifier. <br/> - Verifier fetches the `vp_token` using transaction-id. <br/> - Verifier provides `response_uri` in Authorization request. |
+      | Fragment              | Response will be encoded in the Fragment added to the redirect_uri and will be sent to the verifier when redirecting the end-user back to the verifier                                                                                                                                                                                                     
```

markdownlint-cli2 (0.22.0)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)